ui/app/styles/components/sidebar.scss

```diff
@@ -102,3 +102,32 @@
     }
   }
 }
+// remove above once MenuSidebar component has been removed
+// new sidebar styling
+.brand-icon {
+  .hs-icon {
+    width: 44px;
+    height: 44px;
+  }
+}
+
+.hc-app-frame {
+  font-family: inherit;
+}
+
+.hc-nav-subnav-item-link.hds-button {
+  font-family: inherit;
+}
+
+.hc-nav-container {
+  .popup-menu-content {
+    .menu-label {
+      font-size: 14px;
+      font-weight: $font-weight-bold;
+      text-transform: unset;
+    }
+    .token-alert {
+      padding: 8px;
```

Comment:
this can however be $spacing-xs (comes from the _spacing.scss)

Reply:
Nice find I updated it!